src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -4,6 +4,10 @@ import ConfigStore from '../stores/configStore';
 import ApiMixin from '../mixins/apiMixin';
 import {logAjaxError} from '../utils/logging';
 
+// this is so webpack emits the file into the dist folder. The resulting url links to a superfluous bad file generated by the plugin, which we ignore.
+// eslint-disable-next-line no-unused-vars
+const gif = require('file-loader!../../images/onboarding/release-commits-modal.gif');
```

Comment:
what

I'm definitely deferring to @benvinegar here. This can't be the best way here.